Разметка всех страниц

username1298 opened the pull request:
Дополнительно скорректирован файл ".editorconfig" взятый из htmlacademy/codeguide

---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2300403/tasks/5)

Commit:
выполняет разметку страниц в файлах index.html и catalog.html

Дополнительно скорректирован файл ".editorconfig" взятый из htmlacademy/codeguide

AlexandrRumiantsev commented:
1) header не должен быть внутри мэйна (https://dropmefiles.com/SVKRB)
2) У всех alt и visually-hidden,  конце должны быть точки (https://up.htmlacademy.ru/htmlcss/36/criteries#ALLY-02)
3) Каждый <section> должен иметь заголовок, если нет явного, то  visually-hidden (https://up.htmlacademy.ru/htmlcss/36/criteries#HTML-02) (нет в page-footer-address, special-delivery)
4)Файлы и папки должны быть записаны строчными буквами без пробелов между словами. (https://up.htmlacademy.ru/htmlcss/36/criteries#PROJ-03)

username1298 committed:
Rename Contributing.md to contributing.md

Изменение регистра имени файла по замечанию наставника, согласно правилам (https://up.htmlacademy.ru/profession/frontender/16/htmlcss/36/criteries#PROJ-03)

username1298 committed:
Rename Readme.md to readme.md

Изменение регистра имени файла по замечанию наставника, согласно правилам (https://up.htmlacademy.ru/profession/frontender/16/htmlcss/36/criteries#PROJ-03)

Commit:
Вносит правки согласно замечаниям наставника.

п.1 Сменил header на section, latalog.html строка 103 (изначально проверял валидатором, ошибок первично не выдавал, счел, что
п.2 Во всех alt и visually-hidden проставил точки. У иконок alt оставлен пустым, согласно правилам (https://htmlacademy.ru/blog/html/alt-text)
п.3 Исправил иерархию заголовков. В каждый <section> добавил заголовокh2, там где он не требовался, был добавлен атрибут class="vusially-hidden" (https://up.htmlacademy.ru/profession/frontender/16/htmlcss/36/criteries#HTML-02)
п.4 Файлы в рабочей папке переименованы нижним регистром, несмотря на то, что это дефолтные названия подтянутые с мастера.
